web_company_color/models/res_company.py

```diff
@@ -200,21 +201,27 @@ def _scss_get_sanitized_values(self):
         )
         return values
 
-    def _scss_generate_content(self):
+    def _scss_generate_content(self, uninstall=False):
         self.ensure_one()
         # ir.attachment need files with content to work
         if not self.company_colors:
             return "// No Web Company Color SCSS Content\n"
-        return self.SCSS_TEMPLATE % self._scss_get_sanitized_values()
+        if uninstall:
+            return (
+                self._get_scss_template_uninstall() % self._scss_get_sanitized_values()
+            )
+        return self._get_scss_template() % self._scss_get_sanitized_values()
 
     def scss_get_url(self):
         self.ensure_one()
         return URL_SCSS_GEN_TEMPLATE % self.id
 
-    def scss_create_or_update_attachment(self):
+    def scss_create_or_update_attachment(self, uninstall=False):
         IrAttachmentObj = self.env["ir.attachment"]
         for record in self:
-            datas = base64.b64encode(record._scss_generate_content().encode("utf-8"))
+            datas = base64.b64encode(
+                record._scss_generate_content(uninstall).encode("utf-8")
```

Comment:
**nitpick:** you can get `uninstall` directly from the context inside the `_scss_generate_content` method, these changes could be removed

Reply:
Removed these changes